docs/source/NEWS.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 20.1.2 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- fix ``ModuleNotFoundError: No module named 'cStringIO'`` in ldaptor-ldap2pdns.
+- move scripts to console_scripts entry_points
```

Comment:
This is ok as I don't want to make ldaptor complicated... and I don't know how many people are using ldaptor.

But this release notes text looks like a commit message, rather than an information targeted to ldaptor end users.

The release notes can be something like this:

* The ldaptor scripts were converted to entry points. If you are just calling them scripts from external source nothing was changes. If you import code from the scripts, you will need to update your code as the actual scripts python code was moved into the `ldaptor._scripts` package.

Reply:
I don't think any of the bin files were importable - they all contained hyphens and no `.py` extension

Reply:
OK. I have never used the scripts directly. I tried to use them as example, but they looked complicated  :)

I have used docs/source/examples/quickstart_client.py as a good start.
I have the following script which I find it "easy" to understand: https://gist.github.com/adiroiban/d1926cfc607fd5ef588a908af61d07fd

So maybe the scripts could be redesigned to also server as documentation...
and instead of docs/source/examples/quickstart_client.py , docs/source/examples/client_basic.py and bin/ldaptor-search we can have a single reference script.